scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate nodes with invalid categories."""
+
+import logging
+import sys
+
+from modularodm import Q
+from nose.tools import *
+from tests.base import OsfTestCase
+from tests.factories import UserFactory
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+
+
+def main():
```

Comment:
Is this script related to this PR? Seems to it might be for a different feature.